<?php
require_once 'includes/tutorial_header.inc.php'; ?>

 <div class="content_nav">
	<?php require_once('includes/tutorial_nav.php'); ?>
  
  </div>
  <div class="content_content">
<p>
<p><b><font color='red'>EFI-GNT Input and Output Pages</font></b></p>
<p>The EFI-GNT web tool is a user-friendly interface to the software that accepts the query input SSN, collects the genome neighbors, and generates the colored version of the input SSN, the GNN in the two formats described in the previous section, and text files for download.  The software is run on a server housed in the Institute for Genomic Biology (IGB) at the University of Illinois at Urbana-Champaign.</p>

<p><b><font color='red'>Start Page/Input</font></b></p>

<p><b>The input SSN must be in the form of an xgmml file for 1) a SSN generated by either Option A or Option B of the EFI-EST web tool or 2) a SSN generated by either Option A or Option B from the EFI-EST web too and manipulated and exported from <a href='http://www.cytoscape.org/download.html'>Cytoscape</a>.</b></p>

<p><b>SSNs generated with Option C of the EFI-EST will not work—the process for generating the GNN requires that the sequences have UniProt IDs.</b></p>

<p>The maximum size of the xgmml file is 2048 MB.  The SSN may be either a full SSN (a node for each sequence) or a representative-node (rep-node) SSN (sequences sharing greater than a user-selected sequence identity are located in the same metanode).</p>

<p>EFI-GNT uses a default ± 10 orf window to collect the genome neighbors; the user can select a smaller window (from ± 3 – ± 20 orfs) in the “Neighborhood Size” pull-down menu</p>

<p>EFI-GNT collects all genome neighbors within the specified window.  However, it will display a spoke node only if the query-neighbor co-occurrence frequency is greater than a specified value.  The default value is 20%.  A smaller value, e.g., 5%, should also be used to find neighbors that co-occur with low frequency, often as the result of phylogenetically diverse genome arrangements of functionally linked pathway enzymes.  As the co-occurrence frequency is decreased, a larger number of neighbors and Pfam families will be reported in the GNN.</p>

<p>As with EFI-EST, the user also inputs an e-mail address to which an email containing a link to the results will be sent.</p>

<p><br><img src='new_images/input_figure_1.jpg' alt='Figure 1' width='600'></p>

<p><b><font color='red'>Download Page/Output</font></b></p>

<p>When the results are available (typically a few minutes, although the time required for the analysis increases with the number of query sequences in the input SSN), an e-mail with a link to the output is sent to the address provided by the user on the Start page.  The link will be active for seven days.</p>

<p><br><img src='new_images/input_figure_2.jpg' alt='Figure 2' width='600'></p>

<p>The EFI-GNT output is three xgmml files and several text/spreadsheet files.</p>

<p><b>Colored SSN:</b>  The colored version of the SSN described in the previous section is available for download as an xgmml file for viewing in Cytoscape.  This SSN allows the user to quickly associated SSN cluster spoke nodes in the GNNs with clusters in the input query SSN.</p>

<p><b>Two formats of the GNN:</b>  The two formats of the GNN described in the previous section are available for download as xgmml files and viewing/analysis in Cytoscape:</p>

<p>1.  A cluster is present for each Pfam family (hub-node) that was identified as a neighbor to queries in the SSN clusters (spoke-nodes).  This format allows the user to assess whether queries in multiple SSN clusters are neighbors to members of the same Pfam family and, therefore, may have the same in vitro activities and in vivo metabolic functions.</p>

<p>2.  A cluster is present for each query SSN cluster (hub-node) that was used to identify genome neighbors (spoke-nodes).   This format allows the user to identify functionally linked enzymes, as deduced from genome proximity, that constitute the metabolic pathway in which the sequences in the query SSN cluster participate.</p>

<p><b>Text/Spreadsheet files:</b>  Additional files are available to allow the user to perform additional analyses.  At present these are:</p>

<p>1.  Text file with list of query accession IDs not found in the bacterial and fungal ENA files (nomatch.tab), i.e., not in the STD (annotated assembled sequences), CON (high level constructed sequences), and WGS (whole genome shotgun sequencing with intermediate level of assembly) files for bacterial and fungal proteins.</p>

<p>This can be used to generate custom node attribute to identify sequences with no matches in the query SSN.</p>

<p>2.  Text file with list of query accession IDs that do not have genome neighbors (noneighb.tab), in the bacterial and fungal ENA files, i.e., the ENA files contain single orfs.<p>

<p>This can be used to generate custom node attribute to identify sequences with no neighbors in the query SSN.</p>

<p>In the near future, we will provide additional files to facilitate downstream analyses, including the mapping of neighbors to the SSNs for their Pfam families.</p>




        </div>
<div>
  <div> </div>
      </div>
    </div>

    <div class="clear"></div>

	</div>
    
    
    
    
    
    
   <!-- END: class="content_wide" -->
</div>

<p style='text-align:center;'><a href='tutorial_gnn_manipulation.php'><button class="css_btn_class">Continue Tutorial</button></a></p>

<?php require_once('includes/tutorial_footer.inc.php'); ?>
